app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def show_recent_post
+    posts.where(author_id: id).first(3)
+  end
+
+  after_save :update_post_counter
+
+  private
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] [REQUIRED] Hello @Patfarmurs, this recommendation by the previous reviewer is supposed to be implemented in the Post model. The reviewer gave the inline recommendation on the Post model because the Post model needs to have a method that updates the posts counter for a user. 
Kindly resolve this 👍🏽